pkg/image/types/types.go

```diff
@@ -36,6 +36,8 @@ const (
 	PullTypeNever     PullType = "never"
 )
 
+var DefaultTransport = "oci-archive"
```

Comment:
`const`?

Reply:
I wrote it on purpose. Pass the parameters to modify this value.